nodes/parachain/src/chain_spec/spiritnet.rs

```diff
@@ -164,8 +165,15 @@ pub fn get_chain_spec_wilt() -> Result<ChainSpec, String> {
 				id,
 			)
 		},
-		vec![],
-		None,
+		vec![
+			"/dns4/bootnode.kilt.io/tcp/30360/p2p/12D3KooWRPR7q1Rgwurd4QGyUUbVnN4nXYNVzbLeuhFsd9eXmHJk"
+				.parse()
+				.expect("bootnode address is formatted correctly."),
```

Comment:
Should it be bootnode address is NOT formatted correctly? As the message is shown when `expect()` fails, right?

Reply:
`KILT telemetry url is valid; qed` so the intention is to put the proof in the expect message.

> this will never fail because the URL is valid

Reply:
a87edf7